cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -377,6 +446,22 @@ impl YaccParser {
                 i = self.parse_ws(j, true)?;
                 continue;
             }
+            if let Some(j) = self.lookahead_is("%expect-unused", i) {
+                i = self.parse_ws(j, false)?;
+                while i < self.src.len() {
+                    if self.lookahead_is("%", i).is_some() {
+                        break;
+                    }
+                    let (j, n, span) = self.parse_token(i)?;
```

Comment:
There is `parse_rule` so I think what we really want is something like:

```rust
match self.parse_token(i) {
  Ok(...) => ...,
  Err(_) => {
    match self.parse_rule(i) {
      Ok(...) => ...,
      Err(_) => return Err("Expected token or rule at {i}")
  }
};
```

Reply:
Modulo `s/parse_rule/parse_name/` and inverting the order so rule names are parsed first, I believe this works, pushed in [bef29c0](https://github.com/softdevteam/grmtools/pull/345/commits/bef29c05d0f6628ea314131357342fbe58362fe7)